docs/develop/localization.md

```diff
@@ -145,6 +145,195 @@ For Outlook add-ins, the [SourceLocation] element also aligns to the form factor
+
+- The `name` attribute of the `<Token>` must match the name you used in the token in the `Url` of `<ExtendedOverrides>`.
+- There are other types of tokens, so you must specify `LocaleToken` as the type.
+
+```xml
+    ...
+    </VersionOverrides>  
+    <ExtendedOverrides Url="https://contoso.com/addin/${token.locale}-extended-overrides.json" >
+        <Tokens>
+            <Token xsi:type="LocaleToken" name="locale" default="en-us">
+                <!-- Child elements are described later in the article -->
+            </Token>
+        </Tokens>
+    </ExtendedOverrides>
+</OfficeApp>
+```
+
+Now you need to tell Office what string to insert in the URL when you *do* have an extended overrides file for the Office host application's locale. You do this with a series of [Override](../reference/manifest/override.md) elements. In the following example, there are two extended overrides files (in addition to the one for the default `en-us`). Note that:
+
+- The `Locale` attribute is the one that Office will try to match with the Office host application's locale.
+- The `Value` attribute is the string that Office will insert in the URL when it finds a match.
+
+```xml
+    ...
+    </VersionOverrides>  
+    <ExtendedOverrides Url="https://contoso.com/addin/${token.locale}-extended-overrides.json" >
+        <Tokens>
+            <Token xsi:type="LocaleToken" name="locale" default="en-us">
+                <Override Locale="fr-fr" Value="fr-fr" />
+                <Override Locale="ja-jp" Value="ja-jp" /> >
+            </Token>
+        </Tokens>
+    </ExtendedOverrides>
+</OfficeApp>
+```
+
+You can use the wildcard character, "\*", in the `Locale` attribute. This is useful when you want to insert the same string into the URL for multiple locales. For example, suppose you have an extended overrides file that has be localized to `es-es` (Spanish in Spain) and you want it to be used in all Spanish-speaking countries. Without the wildcard character, you would need a different `<Override>` element for every Spanish locale; `es-es`, `es-mx`, `es-ar`, `es-bo`, etc. as in the following example.
+
+```xml
+<Token xsi:type="LocaleToken" name="locale" default="en-us">
+     ...
+    <Override Locale="es-es" Value="es-es" />
+    <Override Locale="es-mx" Value="es-es" />
+    <Override Locale="es-ar" Value="es-es" />
+    <Override Locale="es-bo" Value="es-es" />
+     ...
+</Token>
+```
+
+But by using the wildcard character, you can have just a single `<Override>` element.
+
+```xml
+<Token xsi:type="LocaleToken" name="locale" default="en-us">
+     ...
+    <Override Locale="es-*" Value="es-es" />
```

Comment:
Is it possible to use just the ll e.g. `es`? If so, would the wildcard work in that instance?

Reply:
No. You can't just use the ll.